src/main/kotlin/dev/arbjerg/lavalink/client/LavalinkClient.kt

```diff
@@ -60,16 +55,12 @@ class LavalinkClient: Closeable {
      * @param region (not currently used) The voice region of your node
      */
     @JvmOverloads
-    fun addNode(name: String, address: URI, password: String, region: VoiceRegion = VoiceRegion.NONE): LavalinkNode {
-        if (userId == -1L) {
-            throw IllegalStateException("User ID not set, please use LavalinkClient#setUserId(Long) to set it before adding nodes.")
-        }
-
```

Comment:
Any reason why you'd remove this?

Reply:
Because the ID is now a constructor parameter.